components/sidebar/sidebar_category_header.tsx

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import classNames from 'classnames';
+import React from 'react';
+import {DraggableProvidedDragHandleProps} from 'react-beautiful-dnd';
+
+import {wrapEmojis} from 'utils/emoji_utils';
+
+type Props = {
+    children?: React.ReactNode;
+    displayName: string;
+    dragHandleProps?: DraggableProvidedDragHandleProps;
+    isDragging?: boolean;
+    isDraggingOver?: boolean;
+}
+
+export const SidebarCategoryHeader = React.forwardRef((props: Props, ref?: React.Ref<HTMLDivElement>) => {
+    return (
+        <div className='SidebarChannelGroupHeader'>
+            <div
+                ref={ref}
+                className={classNames('SidebarChannelGroupHeader_groupButton', {
+                    dragging: props.isDragging,
+                })}
+                aria-label={props.displayName}
+            >
+                <div {...props.dragHandleProps}>
+                    {wrapEmojis(props.displayName)}
+                </div>
```

Comment:
Do we need the `dragHandleProps` code here given it's a static header?

Reply:
Oh static is a much better name for it. I would've removed those both from the static one, but I couldn't come up with a better name than something like `SidebarCategoryHeaderNotCollapsibleOrDraggable` 😛 . Static vs not static is much better though since it also lets the collapsible/draggable one be the primary one